.github/workflows/release.yml

```diff
@@ -206,6 +206,13 @@ jobs:
                 libtool
 
       - name: Build
+        env:
+          # the default Xcode (and macOS SDK) version can be found at
+          # <https://github.com/actions/runner-images/blob/main/images/macos/macos-15-Readme.md#xcode>
+          #
+          # then the minimal supported deployment target of that macOS SDK can be found at
+          # <https://developer.apple.com/support/xcode/#minimum-requirements>
```

Comment:
OK for the change, but why these comments related to xcode?

Reply:
Because macOS SDK is part of Xcode. 

In the first link, you can see GitHub installed multiple versions of Xcode in the runner image, and each macOS SDK version is contained in one or more Xcode versions.

In fact, neither Xcode nor macOS SDK version matter now, because all recent macOS SDK versions support building for OS X 10.13. I just want to document where this 10.13 comes from.